addon/manifests/chart/templates/role_binding.yaml

```diff
@@ -0,0 +1,16 @@
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ template "search.fullname" . }}-pod
```

Comment:
Where does this RoleBinding gets used? I assume this get deployed to the managed cluster, but the RoleBinging for the search-collector is defined in a different file (above). Is there any other pod that uses this one?

Reply:
I copied the full chart from klusterlet-addon-operator https://github.com/stolostron/klusterlet-addon-operator/tree/main/helm-charts/search/templates 
I have the same question too.  if search-collector do not need get/delete pod permission, I will delete it. let me check if there is some historical reason with the previous maintainer.

Reply:
Get permission is already granted by `cluster_role.yaml` and we don't need the delete pod permission. So it's ok to remove this and `role.yaml`.

Reply:
removed.

Reply:
thanks!